<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>132</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>77</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>77</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../icons.qrc">
              <normaloff>:/pic/icons/示意图_训练数据.png</normaloff>:/pic/icons/示意图_训练数据.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>94</width>
              <height>85</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="minimumSize">
             <size>
              <width>77</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>77</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>训练数据</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>77</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>77</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../icons.qrc">
              <normaloff>:/pic/icons/示意图_验证数据.png</normaloff>:/pic/icons/示意图_验证数据.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>94</width>
              <height>85</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="minimumSize">
             <size>
              <width>77</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>77</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>验证数据</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="minimumSize">
              <size>
               <width>39</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>39</width>
               <height>10</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:transparent;
border:None;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../icons.qrc">
               <normaloff>:/pic/icons/示意图_大向右箭头.png</normaloff>:/pic/icons/示意图_大向右箭头.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>39</width>
               <height>10</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QPushButton" name="pushButton_6">
           <property name="minimumSize">
            <size>
             <width>94</width>
             <height>77</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>94</width>
             <height>77</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../icons.qrc">
             <normaloff>:/pic/icons/示意图_代理模型.png</normaloff>:/pic/icons/示意图_代理模型.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>94</width>
             <height>85</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="minimumSize">
              <size>
               <width>39</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>39</width>
               <height>10</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:transparent;
border:None;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../icons.qrc">
               <normaloff>:/pic/icons/示意图_大向右箭头.png</normaloff>:/pic/icons/示意图_大向右箭头.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>39</width>
               <height>10</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_9">
           <property name="minimumSize">
            <size>
             <width>94</width>
             <height>77</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>94</width>
             <height>77</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../icons.qrc">
             <normaloff>:/pic/icons/示意图_拟合模型.png</normaloff>:/pic/icons/示意图_拟合模型.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>94</width>
             <height>85</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="3" rowspan="2">
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>拟合模型</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>代理模型</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>38</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>219</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>220</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>131</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
